src/daemon/rpmostreed-transaction-types.cxx

```diff
@@ -1165,7 +1165,8 @@ deploy_transaction_execute (RpmostreedTransaction *transaction, GCancellable *ca
       && (rpmostree_origin_get_regenerate_initramfs (origin)
           || rpmostree_origin_has_initramfs_etc_files (origin)))
     {
-      rpmostree_origin_set_regenerate_initramfs (origin, FALSE, NULL);
+      auto argsv = util::rust_stringvec_from_strv (NULL);
```

Comment:
The lack of `Option` (https://github.com/dtolnay/cxx/issues/87) is really annoying here. But didn't feel like it was worth adding more APIs for this (e.g. a `rpmostree_origin_disable_initramfs_regenerate`). I'm hoping this work will make these clients easier to port to Rust in the future, which will allow us to clean this up anyway.